test/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit/Model/Kepler/COE.test.cpp

```diff
@@ -289,6 +292,18 @@ TEST_F(OpenSpaceToolkit_Astrodynamics_Trajectory_Orbit_Model_Kepler_COE, GetAngu
     }
 }
 
+TEST_F(OpenSpaceToolkit_Astrodynamics_Trajectory_Orbit_Model_Kepler_COE, ComputeLTAN)
+{
+    {
+        const Angle raan = Angle::Radians(4.289024646407403);
+        const Instant instant = Instant::DateTime(DateTime::Parse("2021-06-19T19:35:35"), Scale::UTC);
+
+        const Real ltan = COE::ComputeLTAN(raan, instant);
+
+        EXPECT_NEAR(22.5, ltan, 1e-1);
```

Comment:
```suggestion
        EXPECT_NEAR(22.55, ltan, 1e-2);
```
Can we increase the assertion precision to two decimal places?